go/vt/vttablet/tabletmanager/rpc_external_reparent.go

```diff
@@ -193,51 +200,112 @@ func (agent *ActionAgent) finalizeTabletExternallyReparented(ctx context.Context
 		return errs.Error()
 	}
 
-	wg.Add(1)
```

Comment:
removing the synchronization here is deliberate. We want to update the shard record first, and only then refresh the old master, because it is possible that the old master tablet is unreachable.

Reply:
Feel free to add this as a comment for posterity even though it's talking about code that doesn't exist anymore. I find it helps to leave notes like "We used to ... but we don't anymore because ...". Actually I think there's an example of one of these comments just below.

Reply:
done.